doc/source/overview/getting_started.md

```diff
@@ -37,12 +37,52 @@ pip install alibi[shap]
 :class-label: sd-pt-0
 :sync: label-dist
 ```{div} sd-mb-1
-Installation with support for distributed computation of explanations.
+Installation with support for AnchorTabularDistributed and 
+[distributed Kernel SHAP](../examples/distributed_kernel_shap_adult_lr.ipynb).
 ```
 ```bash
 pip install alibi[ray]
 ```
 ````
+
+````{tab-item} TensorFlow
+:class-label: sd-pt-0
+:sync: label-tensorflow
+```{div} sd-mb-1
```

Comment:
Is it possible to have a bullet list to enhance readability? 
![image](https://user-images.githubusercontent.com/13080878/162750047-62d85436-a3b2-4398-bfa6-632954622204.png)